SampleApps/TestApp/AppDelegate.swift

```diff
@@ -22,7 +22,7 @@ import UIKit
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
     // TODO: Set up the preferred Environment File ID from your mobile property configured in Data Collection UI
-    private let ENVIRONMENT_FILE_ID = ""
+    private let ENVIRONMENT_FILE_ID = "yourAppId"
```

Comment:
I'd suggest just leaving this as an empty string.  If the ID isn't changed, then Configuration will persist "yourAppId" and attempt to remote download the configuration. If the ID is just an empty string, the Configuration call is aborted  without making a network call.

Reply:
Didn't think  about  that.  We can leave it as an empty string.